packages/astro/src/@types/astro.ts

```diff
@@ -444,15 +444,22 @@ export interface AstroUserConfig {
 	 * @name base
 	 * @type {string}
 	 * @description
-	 * The base path to deploy to. Astro will build your pages and assets using this path as the root. Currently, this has no effect during development.
+	 * The base path you're deploying to. Astro will match this pathname during development so that your development experience matches your build environment as closely as possible. You can access this value in your app via `import.meta.env.BASE_URL`.
 	 *
-	 * You can access this value in your app via `import.meta.env.BASE_URL`.
+	 * In the example below, `astro dev` will start your server at `/docs`.
 	 *
 	 * ```js
 	 * {
 	 *   base: '/docs'
 	 * }
 	 * ```
+	 * 
+	 * When using this option, all of your static asset imports and URLs should add the base as a prefix.
+	 * 
+	 * ```astro
+	 * <a href="/docs/about/>About</a>
+	 * <img src="/docs/image.png">
```

Comment:
Maybe show how we'd write URLs that adapt to changing base URLs?
```suggestion
	 * <a href="{import.meta.env.BASE_URL}/about/">About</a>
	 * <img src="{import.meta.env.BASE_URL}/image.png">
```

Reply:
Should we show this *instead* of using the base from the example? Or in addition to?

Reply:
I'd thought the literal base was clear enough, but it might not be. You could do one of each?

Reply:
Personally I think `/docs/` is clear enough, but using `` `${import.meta.env.BASE_URL}/image.png` `` for the second example couldn't hurt.